content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -11,11 +11,24 @@ This page shows how to configure a Key Management Service (KMS) provider and plu
 
 * {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1" %}}
 * Kubernetes version 1.10.0 or later is required
 
 * etcd v3 or later is required
 
 {{< feature-state for_k8s_version="v1.12" state="beta" >}}
+{{< /tab >}}
```

Comment:
Should this be `{{% /tab %}}`?
To be honest, I don't like tabs. The value it brings is very limited and the complexity of the document structure is increased. I don't know how such a page can be converted into a PDF file, for example.

Reply:
Good catch!

We need this doc to support both v1beta1 and v1alpha1 for awhile. If not tab, do you have other suggestions?

Reply:
The using of tabs is a personal taste. I'd prefer laying out the options one by one, in sub-sections. I can skip sub-sections quickly if it is not relevant to me. This is not a blocker.

Reply:
updated. PTAL